docs/plugins/repository-azure.asciidoc

```diff
@@ -64,8 +64,8 @@ cloud:
 
 `my_account1` is the default account which will be used by a repository unless you set an explicit one.
 
-You can set the timeout to use when making any single request. It can be defined globally, per account or both.
-Defaults to `5m`.
+You can set the client side timeout to use when making any single request. It can be defined globally, per account or both.
+It's not set by default which means that elasticsearch is using the default value set by the azure client.
```

Comment:
I did not want to do it because if azure changes it at some point our documentation will become inconsistent... thoughts?